src/System.Data.Common/tests/System/Data/DataColumnTest2.cs

```diff
@@ -395,38 +364,18 @@ public void Unique_PrimaryKey()
 
             Assert.True(col.Unique);
 
-            try
-            {
-                col.Unique = false;
-                Assert.False(true);
-            }
-            catch (ArgumentException e)
-            {
-            }
+            Assert.Throws<ArgumentException>(() => col.Unique = false);
 
             Assert.True(col.Unique);
         }
 
-        [Fact]
-        public void ctor()
-        {
-            DataColumn dc;
-            dc = new DataColumn();
-
-            // ctor
-            Assert.False(dc == null);
```

Comment:
`dc` can't be null unless the ctor throws, but then it's not expected to throw & the test will fail anyway if thrown.